narwhals/dataframe.py

```diff
@@ -195,17 +195,28 @@ class DataFrame(BaseFrame):
     Examples:
         Constructing a DataFrame from a dictionary:
 
+        >>> import pandas as pd
         >>> import polars as pl
         >>> import narwhals as nw
         >>> data = {"a": [1, 2], "b": [3, 4]}
         >>> df_pl = pl.DataFrame(data)
-        >>> df = nw.DataFrame(df_pl)
+        >>> df_pd = pd.DataFrame(data)
+        >>> df = nw.from_native(df_pl)
         >>> df
```

Comment:
No need to print the narwhals dataframe here, we can instead pass the other dataframes to the function like below to print actual dataframes.

Reply:
Should I take out line 205 to 209?